### ISSUE TITLE
Pass info from FEEDS to ItemExporter

### ISSUE THREAD

**Issue by StasDeep** [scrapy/scrapy#4606]
## Summary

The new FeedExporter class doesn't allow to pass custom info to ItemExporter through the `FEEDS` setting.

## Motivation

My use-case is a bit too long to describe, but it can be minimized to the following: for example, I have a custom ItemExporter which I want to have different options for different feeds. Say I want to pass custom `param` option to it.

```python
class CustomItemExporter(BaseItemExporter):
    
    def _configure(self, options, dont_fail=False):
        super()._configure(options, dont_fail)
        self.param = options.pop('param', None)
        
    def export_item(self, item):
        # Using self.param here
        ...
```

Ideally, I would like to do set it like this:

```python
FEEDS = {
    uri_detail: {'format': 'custom', 'item_export_kwargs': {'param' : 'Detail'}},
    uri_overview: {'format': 'custom', 'item_export_kwargs': {'param' : 'Overview'} }
}
```

## Describe alternatives you've considered

The only workaround I've found was to create a new FeedExporter class and override `open_spider` method to pass `feed['item_export_kwargs']` to `_get_exporter` method.

## Additional context

Will provide a PR this week.

**Comment by StasDeep** [comment]
I will provide a PR this week.

**Comment by Gallaecio** [comment]
I implemented this in #4512, as I needed it there.

**Comment by StasDeep** [comment]
@Gallaecio if I understand the changes correctly, it is now possible to pass it to FeedStorage. What I want, though, is passing custom args to ItemExporters.
